data_pipeline/elife_article_xml/elife_article_xml_pipeline.py

```diff
@@ -29,8 +29,16 @@ class GitHubRateLimitError(requests.RequestException):
     pass
 
 
-def get_json_response_from_url(url: str) -> Any:
-    response = requests.get(url=url)
+def get_json_response_from_url(
+    url: str,
+    headers: Mapping[str, str] = None,
+) -> Any:
+    if headers:
+        LOGGER.info("API call with access token...")
```

Comment:
Maybe nitpicking, but here we just know whether headers have been configured, not whether it using an access token. If you pass in the `printable_mapping`, then we could log it. (Also together with the URL would probably be useful)